src/main/java/org/jvnet/hudson/test/junit/jupiter/InboundAgentExtension.java

```diff
@@ -669,7 +669,7 @@ private static Slave createAgentJR(JenkinsRule r, Options options)
         s.setLabelString(options.getLabel());
         s.setRetentionStrategy(RetentionStrategy.NOOP);
         r.jenkins.addNode(s);
-        // SlaveComputer#_connect runs asynchronously. Wait for it to finish for a more deterministic test.
+        // AgentComputer#_connect runs asynchronously. Wait for it to finish for a more deterministic test.
```

Comment:
No Java class named `AgentComputer`
```suggestion
        // SlaveComputer#_connect runs asynchronously. Wait for it to finish for a more deterministic test.
```